### PR TITLE
copy dynamic import chunks to functions dirs

### PR DESCRIPTION
**Pull request by lindsaylevine**
Fixes https://github.com/netlify/next-on-netlify/issues/121

### CODE REVIEW

File: lib/helpers/setupNetlifyFunctionForPage.js [hunk 1]
```diff
@@ -47,8 +48,11 @@ const setupNetlifyFunctionForPage = ({
     });
   });
 
+  // Copy any dynamic import chunks
+  copyDynamicImportChunks(functionDirectory);
+
   // Copy page
-  const nextPageCopyPath = join(functionDirectory, "nextPage.js");
+  const nextPageCopyPath = join(functionDirectory, "nextPage", "index.js");
```
Comment (erezrokah) [inline lib/helpers/setupNetlifyFunctionForPage.js:55]: The reason behind this change is to organize all the page files in the same directory, or is it also required for the fix to work?

File: lib/helpers/copyDynamicImportChunks.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const { join } = require("path");
+const { copySync, readdirSync } = require("fs-extra");
+const { logTitle } = require("../helpers/logger");
+const { NEXT_DIST_DIR } = require("../config");
+
+// Check if there are dynamic import chunks and copy to the necessary function dir
+const copyDynamicImportChunks = (functionPath) => {
```
Comment (erezrokah) [inline lib/helpers/copyDynamicImportChunks.js:7]: Does this mean we're copying all `chunks` to all functions?

I was also worried users can control the `chunks` configuration until I found [this](https://github.com/vercel/next.js/issues/8834)

**Review by erezrokah — APPROVED** [review]
@lindsaylevine, this looks good as a fix for the issue.

You can discard my comments if you feel they are not relevant

File: lib/helpers/copyDynamicImportChunks.js (hunk 2 above)
Comment (lindsaylevine) [reply lib/helpers/copyDynamicImportChunks.js:7]: yeah, it does. i think it should be okay for now and can definitely revisit later if issues arise from this implementation. reading the lambdas for the chunk names should be an "if absolutely necessary" approach imo 😂

File: lib/helpers/setupNetlifyFunctionForPage.js (hunk 1 above)
Comment (lindsaylevine) [reply lib/helpers/setupNetlifyFunctionForPage.js:55]: yeah, what jason said :( the lambda generated by next (that we previous renamed to nextPage.js to pair with next_whatever_the_page_name_is.js) is searching for the chunks one level up at `../` 💀

**Pull request merged**